documentation/api.tex

```diff
@@ -0,0 +1,69 @@
+\chapter{API documentation}
+This chapter will cover the documentation of the back-end API.
+
+\section{GraphiQL}
```

Comment:
This is more of a how-to than a documentation in my opinion.
To me it could just say:

GraphiQL is available on URL bla on the backend.